components/LeagueEntries/LeagueEntries.interface.ts

```diff
@@ -6,4 +6,5 @@ export interface ILeagueEntriesProps {
   linkUrl: string;
   isEliminated?: boolean;
   isPickSet?: boolean;
-}
\ No newline at end of file
+  teamLogo?: string;
```

Comment:
Make this requiered.

Reply:
Removed the question mark to make it required.